Jenkinsfile.release

```diff
@@ -172,5 +175,33 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
                 }
             }
         }
+
+        if (ostree_prod_refs.size() > 0) {
+            stage("OSTree Import: Wait and Verify") {
+                def tmpd = shwrapCapture("mktemp -d")
+
+                shwrap("""
+                cd ${tmpd}
+                ostree init --mode=archive --repo=.
+                cat /etc/ostree/remotes.d/fedora.conf >> config
+                """)
+
+                timeout(time: 20, unit: 'MINUTES') {
+                    for (ref in ostree_prod_refs) {
+                        shwrap("""
+                        cd ${tmpd}
+                        while true; do
+                            ostree pull --commit-metadata-only fedora:${ref.key}
```

Comment:
might be worth mentioning here in a comment that if there is no signature or the signature is invalid then the pull will fail (correct?) which is why we can rely on this and remove the manual step from the release checklist

Reply:
This was implicit a bit in the commit message, but I mentioned it in the code and made it more explicit in the commit message too.